### DISCUSSION TITLE
Find a hidden word

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#776] [Galaxy Trivia] [answered]
Work out the missing letters needed to fill the highlighted squares. Each letter completes the word on its left and begins the word on its right. When the puzzle is complete, the letters in the coloured column will spell a word, reading from top to bottom.

<img width="455" alt="image" src="https://github.com/ParanoidUser/yolo/assets/5120290/7e297627-5c89-4790-8788-ef33b24115b7">

**Comment by peterbaccaro** [comment, marked as answer]
![image](https://github.com/ParanoidUser/yolo/assets/74666079/93df754d-f109-4e36-afbc-04074cba9ed7)
